src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -256,14 +362,15 @@ check_in(crt_group_t *remote_group, int rank, int tag)
 	server_ep.ep_grp = remote_group;
 	server_ep.ep_rank = rank;
 	server_ep.ep_tag = tag;
+
 	rc = crt_req_create(test_g.t_crt_ctx[0], &server_ep,
 			    TEST_OPC_CHECKIN, &rpc_req);
 	D_ASSERTF(rc == 0 && rpc_req != NULL, "crt_req_create() failed,"
 		  " rc: %d rpc_req: %p\n", rc, rpc_req);
 
 	rpc_req_input = crt_req_get(rpc_req);
 	D_ASSERTF(rpc_req_input != NULL, "crt_req_get() failed."
-		  " rpc_req_input: %p\n", rpc_req_input);
+ 		  " rpc_req_input: %p\n", rpc_req_input);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs